bin/test_sequencer_solo

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
```

Comment:
I'd prefer a name like "test_itemized" or something to 1) avoid the test_sequencer prefix, and then 2) something a bit more descriptive than _solo.

Reply:
renamed to test_itemized